tests/test_icml_conference.py

```diff
@@ -4372,13 +4372,19 @@ def test_meta_review_stage(self, client, openreview_client, helpers):
         venue.create_meta_review_stage()
         helpers.await_queue_edit(openreview_client, 'ICML.cc/2023/Conference/-/Position_Paper_Meta_Review-0-1', count=1)
 
-        assert len(openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Position_Paper_Meta_Review')) == 50
+        invitations = openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Position_Paper_Meta_Review')
+        assert len(invitations) == 50
+
         invitation = openreview_client.get_invitation('ICML.cc/2023/Conference/Submission2/-/Meta_Review')
         assert invitation
-        assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission4/-/Meta_Review')
+        assert invitation.edit['note']['id']['param']['withInvitation'] == invitations[0].id
         assert 'metareview' in invitation.edit['note']['content']
         assert 'suggestions' not in invitation.edit['note']['content']
 
+        invitations = openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Meta_Review_SAC_Revision')
+        assert len(invitations) == 2
```

Comment:
Wouldn't there be 50 `Meta_Review_SAC_Revision` invitations and 50 `Position_Paper_Meta_Review_SAC_Revision` invitations?